### ISSUE TITLE
Multi-Class Metrics Made Simple, Part I: Precision and Recall

### ISSUE THREAD

**Issue by nkumar115** [nkumar115/Data-Science#5]
**TL;DR**
Performance metrics for precision and recall in multi-class classification

**Article Link**

https://towardsdatascience.com/multi-class-metrics-made-simple-part-i-precision-and-recall-9250280bddc2

**Source**
Medium

**Key Takeaways**
What is more important, precision or recall? 

- where a classifier needs to detect diabetes in human patients: Positive” means the patient has diabetes. “Negative” means that the patient is healthy. We want to make sure that Our  classifier has high recall, so that as many diabetics as possible are correctly detected.

- we are building a video recommendation system, and our classifier predicts Positive for a relevant video and Negative for non-relevant video. We want to make sure that almost all of the recommended videos are relevant to the user, so we want high precision.

**Useful Snippets**

**Useful Tools**
Python
**metrics.classification_report(y_true, y_pred, digits=3)**
